.github/workflows/integration-tests.yml

```diff
@@ -130,22 +131,23 @@ jobs:
         kubectl logs deployment/kuberpult-frontend-service
         # We need to exit with error code because "continue-on-errors" bypasses the error and the build is considered successful
         exit 1   
-    - name: Run docker compose
-      run: |
-        docker-compose -f ./docker-compose.tpl.yml up -d
-    - name: Integration test docker-compose
-      id: integrationTests
-      continue-on-error: true
-      run: |
-        curl --retry 3 --retry-max-time 20 --retry-all-errors http://localhost:8081/health
-        curl --retry 3 --retry-max-time 20 --retry-all-errors http://localhost:8080/health
-    - name: Print container logs
-      if: steps.integrationTests.outcome == 'failure'
-      run: |
-        echo "CD logs"
-        docker logs kuberpult-cd-service
-        echo "Frontend logs"
-        docker logs kuberpult-frontend-service
-
-        # We need to exit with error code because "continue-on-errors" bypasses the error and the build is considered successful
-        exit 1
+#    - name: Run docker compose
+#      run: |
+#        docker-compose -f ./docker-compose.tpl.yml up -d
+#    - name: Integration test docker-compose
+#      id: integrationTests
+#      continue-on-error: true
+#      run: |
+#        curl --retry 3 --retry-max-time 20 --retry-all-errors http://localhost:8081/health
+#        curl --retry 3 --retry-max-time 20 --retry-all-errors http://localhost:8080/health
+#        make -C tests/integration-tests test
+#    - name: Print container logs
+#      if: steps.integrationTests.outcome == 'failure'
+#      run: |
+#        echo "CD logs"
+#        docker logs kuberpult-cd-service
+#        echo "Frontend logs"
+#        docker logs kuberpult-frontend-service
+#
+#        # We need to exit with error code because "continue-on-errors" bypasses the error and the build is considered successful
+#        exit 1
```

Comment:
Good idea. I think we can remove this now entirely!

Reply:
done!